armi/operators/tests/test_operators.py

```diff
@@ -183,7 +187,13 @@ def test_setStateToDefault(self):
 
     def test_snapshotRequest(self):
         with TemporaryDirectoryChanger():
+            fluxFile = "rzmflxYA"
+            self.o.cs[CONF_CROSS_SECTION]["YA"].fluxFileLocation = os.path.join(
+                THIS_DIR, "../../physics/neutronics/tests", fluxFile
+            )
```

Comment:
stealing an empty file from elsewhere rather than adding one to this test dir. probably preferable to do the latter?